### ISSUE TITLE
Tar: Permission denied error during build.

### ISSUE THREAD

**Issue by bryan-lunt** [singularityhub/docker2singularity#87]
When I try to convert the image `uiuccs484parallelprog/cs484_student:latest` from singularity hub, I get this cryptic message:

`tar: can't open 'opt/rh/devtoolset-7/enable': Permission denied`

I have no idea how to fix this. I remember there being something about the `tar` binary in this image having problems, but I think this is different, and that was long ago.

I don't even know how to investigate the cause of this.

**Comment by vsoch** [comment]
If the image is built with an older version of singularity (does singularity inspect work?) then I suspect there will be issues. What I would try first is to go back to an earlier release of Docker2singularity, preferably one that matches the version the container was build with (likely around 2.5.1). If that doesn't work, then we will need to debug this specific container - I can't seem to pull it (is your tag / name correct?) Anyway - first try maybe this branch https://github.com/singularityhub/docker2singularity/tree/v2.6. And remember the containers are also available on Docker Hub (and now) Quay.io so you can pull without needing to rebuild. Let me know how that goes!

**Comment by bryan-lunt** [comment]
My image should be this one:

https://hub.docker.com/r/uiuccs484parallelprog/cs484_student

I'll check about the docker version issue.
When I built my own container from scratch with singularity, I was able to use that to convert my image, so at least my immediate needs are met.

**Comment by vsoch** [comment]
Yes, it's definitely the case that docker2singularity was developed long before that conversion was possible. Please post an update when you try the older versions, and glad that your needs are met with Singularity!
